File: JsonData/GetListBookListRequest.java
package com.skymobi.payment.nvl.api.model;

import com.skymobi.payment.nvl.api.model.common.BaseRequest;

public class GetListBookListRequest extends BaseRequest {

	//书单ID
	private long listId;
	
	//列表起始位置
	private int start;
	
	//列表每页条数
	private int limit;



	public long getListId() {
		return listId;
	}

	public void setListId(long listId) {
		this.listId = listId;
	}

	public int getStart() {
		return start;
	}

	public void setStart(int start) {
		this.start = start;
	}

	public int getLimit() {
		return limit;
	}

	public void setLimit(int limit) {
		this.limit = limit;
	}
	
	
}

File: App/JSCode/Protocol/GetListBookListRequest.js



/**
 * @author bangwen.lei
 */
import BaseRequest from './BaseRequest'
export default class GetListBookListRequest extends BaseRequest {
    //书单ID
    listId;    // private long 

    //列表起始位置
    start; //private int 

    //列表每页条数
    limit; // private int
}

